apps/cli/src/node/devel/docker-compose.yml

```
@@ -0,0 +1,150 @@
+version: "3.9"
+
+x-database-config: &database-config
+    POSTGRES_HOSTNAME: localhost
+    POSTGRES_PORT: "5432"
+    POSTGRES_USER: postgres
+    POSTGRES_PASSWORD: postgres
+    POSTGRES_DB: postgres
+
+services:
+    anvil:
+        image: sunodo/anvil
+        environment:
+            ANVIL_IP_ADDR: 0.0.0.0
+        ports:
+            - "8545:8545"
```

```suggestion
            - "8545"
```

This way the host's port will be dynamic.

---

It's very incovenient to have a dynamic host port. This will typically be configured in the user metamask. It's a standard to be at http://localhost:8545

---

We could print the port defined dinamcally, and also add an option to define the port from the `sunodo run` command.

---

I still think the default should be `8545`, not the dynamic one.

---

A second `sunodo run` would result in an error. 

I prefer defaulting to a dynamic and free of conflict, and requiring the port number deifinition an explicit option.

---

Very unusual to run more than one application at the same time during development. It will be very confusing to not default to 8545 and the configuration of metamask.